ansible/roles/virtualbmc-daemon/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Set service name
+  set_fact:
+    service: vbmcd
```

Comment:
For constants like this you can also use a vars/main.yml file in the role.

Reply:
True.